pkg/webhook/server.go

```diff
@@ -187,13 +187,17 @@ func (s *Server) Handle(pattern string, handler http.Handler) {
 
 var _ manager.Runnable = &Server{}
 
-// Start runs the server if s.Dryrun is false.
-// Otherwise, it will print the objects in yaml format.
+// Start runs the server.
+// It will install the webhook related resources depend on the server configuration.
 func (s *Server) Start(stop <-chan struct{}) error {
-	err := s.installWebhookConfig()
-	// if encounter an error or it's in dryrun mode, return.
-	if err != nil || s.Dryrun {
-		return err
+	if s.EnableInstaller {
+		log.Info("webhook installer is enabled")
```

Comment:
We can improve the log --> installing webhook configuration in cluster ? or something for user-friendly.